yt/frontends/open_pmd/tests/test_outputs.py

```diff
@@ -198,39 +133,20 @@ def test_no_particles_out():
 @requires_file(groupBased)
 def test_groupBased_out():
     dss = load(groupBased)
-    field_list = [('all', 'particle_charge'),
-                  ('all', 'particle_mass'),
-                  ('all', 'particle_momentum_x'),
-                  ('all', 'particle_momentum_y'),
-                  ('all', 'particle_momentum_z'),
-                  ('all', 'particle_positionCoarse_x'),
-                  ('all', 'particle_positionCoarse_y'),
-                  ('all', 'particle_positionCoarse_z'),
-                  ('all', 'particle_positionOffset_x'),
-                  ('all', 'particle_positionOffset_y'),
-                  ('all', 'particle_positionOffset_z'),
-                  ('all', 'particle_weighting'),
-                  ('io', 'particle_charge'),
-                  ('io', 'particle_mass'),
-                  ('io', 'particle_momentum_x'),
-                  ('io', 'particle_momentum_y'),
-                  ('io', 'particle_momentum_z'),
-                  ('io', 'particle_positionCoarse_x'),
-                  ('io', 'particle_positionCoarse_y'),
-                  ('io', 'particle_positionCoarse_z'),
-                  ('io', 'particle_positionOffset_x'),
-                  ('io', 'particle_positionOffset_y'),
-                  ('io', 'particle_positionOffset_z'),
-                  ('io', 'particle_weighting'),
-                  ('openPMD', 'J_x'),
-                  ('openPMD', 'J_y'),
-                  ('openPMD', 'J_z'),
-                  ('openPMD', 'e-chargeDensity')]
+    particle_types = ("all", "io", "nbody")
+    field_list = list(product(particle_types, particle_fields))
+    field_list += [
+        ('openPMD', 'J_x'),
+        ('openPMD', 'J_y'),
+        ('openPMD', 'J_z'),
+        ('openPMD', 'e-chargeDensity')
+    ]
     domain_dimensions = [32, 64, 64] * np.ones_like(dss[0].domain_dimensions)
     domain_width = [0.0002752, 0.0005504, 0.0005504] * np.ones_like(dss[0].domain_left_edge)
 
     assert_equal(len(dss), 101)
-    for ds in dss:
+    for i in range(0, len(dss), 20):  # Test only every 20th ds out of the series
```

Comment:
Is the reason for one out of every 20th to speed up testing? or something else?

Reply:
Yes, speed. Since that test doesn't do anything different for an individual ds right now, there's really no point in doing it for each and everyone.

Reply:
Makes sense!